components/Banner/tests/unit/Theming/MDCBannerViewThemingTests.m

```diff
@@ -0,0 +1,135 @@
+// Copyright 2019-present the Material Components for iOS authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import <XCTest/XCTest.h>
+
+#import "MaterialBanner+Theming.h"
+#import "MaterialBanner.h"
+#import "MaterialContainerScheme.h"
+
+static CGFloat const kBannerViewTextViewOpacityDefault = (CGFloat)0.87;
+static CGFloat const kBannerViewDividerOpacityDefault = (CGFloat)0.12;
+
+/**
+ This class confirms behavior of @c MDCBannerView.
+ */
+@interface MDCBannerViewThemingTests : XCTestCase
+
+@property(nonatomic, strong, nullable) MDCBannerView *bannerView;
+@property(nonatomic, strong, nullable) MDCContainerScheme *containerScheme;
+
+@end
+
+@implementation MDCBannerViewThemingTests
```

Comment:
Please add tests that the fonts applied are pre-scaled for Dynamic Type when `mdc_adjustsFontForContentSizeCategory` or `useCurrentContentSizeCategoryWhenApplied` are `YES`.

Reply:
Addressed in 2b214d9dc5b4ea25c72db13d98d31ec020e5142b